catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityFilterPlugin.java

```diff
@@ -16,71 +16,33 @@
 import static ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin.VALIDATION_ERRORS;
 import static ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin.VALIDATION_WARNINGS;
 
+import java.io.Serializable;
 import java.util.Arrays;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.stream.Collectors;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import ddf.catalog.data.Attribute;
 import ddf.catalog.data.Metacard;
 import ddf.catalog.data.Result;
-import ddf.catalog.data.impl.AttributeImpl;
-import ddf.catalog.operation.QueryResponse;
-import ddf.catalog.plugin.PluginExecutionException;
-import ddf.catalog.plugin.PostQueryPlugin;
+import ddf.catalog.operation.Query;
+import ddf.catalog.plugin.PolicyPlugin;
+import ddf.catalog.plugin.PolicyResponse;
 import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.plugin.impl.PolicyResponseImpl;
 
-public class MetacardValidityFilterPlugin implements PostQueryPlugin {
+public class MetacardValidityFilterPlugin implements PolicyPlugin {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(MetacardValidityFilterPlugin.class);
+    private static final Logger LOGGER =
```

Comment:
Showing up as never used now as the only place it was used has been removed. Should we remove?